.circleci/continue_config.yml

```diff
@@ -196,32 +196,32 @@ jobs:
       - store_artifacts:
           path: /tmp/airflow_logs
 
-  engine_adapter_docker_tests:
-    machine:
-      image: ubuntu-2204:2022.10.2
-      docker_layer_caching: true
-    resource_class: large
-    environment:
-      PYTEST_XDIST_AUTO_NUM_WORKERS: 8
-      SQLMESH__DISABLE_ANONYMIZED_ANALYTICS: "1"
-    steps:
-      - checkout
-      - run:
-          name: Install pg_config
-          command: sudo apt-get update && sudo apt-get install libpq-dev
-      - run:
-          name: Install dependencies
-          command: make install-engine-test
-      - run:
-          name: Bring up Dockerized Engines
-          command: make engine-up
-      - run:
-          name: Make sure DBs are ready
-          command: sleep 60
-      - run:
-          name: Run tests
-          command: make engine-docker-test
-          no_output_timeout: 30m
+  # engine_adapter_docker_tests:
```

Comment:
Do we want to leave this commented out?

Reply:
Sorry, I ignored the fact that it was WIP

Reply:
Eventually there wont be big commented blocks but i'm still messing around with stuff at the moment :)